contrib/clojure-package/examples/bert/src/bert/bert_sentence_classification.clj

```diff
@@ -138,14 +162,16 @@
                                         {:label {label-desc (ndarray/array labels [train-num]
                                                                            {:ctx dev})}
                                          :data-batch-size batch-size})
-        model (m/module model-sym {:contexts [dev]
-                                   :data-names ["data0" "data1" "data2"]})]
-    (m/fit model {:train-data train-data  :num-epoch num-epoch
-                  :fit-params (m/fit-params {:allow-missing true
-                                             :arg-params (m/arg-params bert-base)
-                                             :aux-params (m/aux-params bert-base)
-                                             :optimizer (optimizer/adam {:learning-rate 5e-6 :episilon 1e-9})
-                                             :batch-end-callback (callback/speedometer batch-size 1)})})))
+        fitted-model (m/fit (m/module model-sym {:contexts [dev]
+                                                 :data-names ["data0" "data1" "data2"]})
+                            {:train-data train-data  :num-epoch num-epoch
+                             :fit-params (m/fit-params {:allow-missing true
+                                                        :arg-params (m/arg-params bert-base)
+                                                        :aux-params (m/aux-params bert-base)
+                                                        :optimizer (optimizer/adam {:learning-rate 5e-6 :epsilon 1e-9})
+                                                        :batch-end-callback (callback/speedometer batch-size 1)})})]
+    (m/save-checkpoint fitted-model {:prefix fine-tuned-prefix :epoch num-epoch})
```

Comment:
Why do we save the model to disk now? Could we pass in a parameter to the function to to it? This function seems to do too many things?

Reply:
We have to save the model to disk because there's no other way but saving to disk and loading it back in to get a prediction out of it.

Reply:
With the `infer` API I suppose? Maybe we could change this at some point @kedarbellare?